webapp/src/ts/modules/contacts/contacts.component.ts

```diff
@@ -443,6 +442,7 @@ export class ContactsComponent implements OnInit, AfterViewInit, OnDestroy {
         console.error('Error loading contacts', err);
```

Comment:
Since it's setting ` this.loading = false;` in the `finally`. You can remove the line 441 too. 
Same with contacts.